src/app/config/sgid.ts

```diff
@@ -0,0 +1,63 @@
+import { Schema } from 'convict'
+
+import { ISgidVarsSchema } from '../../types'
+
+const HOUR_IN_MILLIS = 1000 * 60 * 60
+const DAY_IN_MILLIS = 24 * HOUR_IN_MILLIS
+
+export const sgidVarsSchema: Schema<ISgidVarsSchema> = {
+  endpoint: {
+    doc: 'The sgID OIDC endpoint',
+    format: '*',
```

Comment:
should the format be `url` instead? should we also set the default to `null` for clarity